libraries/aws_lambda_event_source_mapping.rb

```diff
@@ -30,6 +30,11 @@ def uuid
     @res[:uuid]
   end
 
+  def resource_id
+    return '' unless exists?
```

Comment:
I would return nil rather

Reply:
Sathish as discussed with Clinton he asked to return nil earlier I was using nil only.